catalog/package.json

```diff
@@ -151,6 +151,7 @@
     "@graphql-codegen/typed-document-node": "^1.18.10",
     "@graphql-codegen/typescript": "^1.23.0",
     "@graphql-codegen/typescript-operations": "^1.18.4",
+    "@graphql-codegen/typescript-react-apollo": "^4.3.2",
```

Comment:
This dependency is not used from what I can see, so, you can remove it

Reply:
removed